package.json

```diff
@@ -44,5 +45,6 @@
       "fast-xml-parser@<4.4.1": ">=4.4.1",
       "@75lb/deep-merge@<1.1.2": ">=1.1.2"
     }
-  }
+  },
+  "packageManager": "pnpm@9.6.0+sha256.dae0f7e822c56b20979bb5965e3b73b8bdabb6b8b8ef121da6d857508599ca35"
```

Comment:
For my learning: why is this needed? Isn't `hermit` (not that I love it) what the team uses for the same purpose assuming I guessed the intent correctly?

Reply:
this tells contributors what packageManager we're using. its an industry standard around corepack which is what hermit is using

Reply:
You had me at "industry standard", does this mean we need to remember to manually update this line every time we run hermit update pnpm? If so, you'd loose me again.

Reply:
nope i didn't manually put this there it's put there by default by corepack

Reply:
pnpm would be upgraded by running `corepack up`. if we're somehow sidestepping that right now then yes this line would not be kept updated. but then i don't understand why we have corepack setup in the repo

Reply:
I have not had to run a `corepack` command, and not in favor of needing to run it to keep the newly added line up to date unless base on my understanding. Are you proposing that we use corepack instead of hermit? If not, suggesting to remove this.

Happy to jump in a call to be convinced otherwise.

Reply:
like I explained, hermit is using corepack. so this is functioning as intended. if we don't want corepack we can uninstall it.

Reply:
updated PR to remove corepack

Reply:
Thanks for removing `corepack`. I have no idea why it was included in `hermit` and if we need it or not. My comment is purely about the line added in the PR, which seemed unnecessary because we never had it and things seems to worked "fine"....